ci-operator/step-registry/sandboxed-containers-operator/gather-must-gather/sandboxed-containers-operator-gather-must-gather-ref.yaml

```diff
@@ -0,0 +1,30 @@
+ref:
+  as: sandboxed-containers-operator-gather-must-gather
+  best_effort: true
+  optional_on_success: false
+  from: cli
+  commands: sandboxed-containers-operator-gather-must-gather-commands.sh
+  env:
+  - name: ENABLE_MUST_GATHER
+    default: "true"
+    documentation: |-
+      Whether to run the must-gather step. Set to "false" to skip must-gather collection.
+  - name: MUST_GATHER_ON_FAILURE_ONLY
+    default: "false"
```

Comment:
TBH I'd prefer the default to be "true" so our configuration would usually be empty (at least that's what I assume we want most of the time, right?)